recipe/recipe.yaml

```
@@ -23,7 +23,7 @@ requirements:
     - pip
```

https://github.com/conda-forge/sqlalchemy-hana-feedstock/pull/14#issuecomment-3421681510

```suggestion
    - pip
    - setuptools
```